### PR TITLE
管理者画面の注文一覧機能の作成

### PR DESCRIPTION
**Pull request by tkeshun**

### CODE REVIEW

File: src/main/java/com/example/controller/AdminController.java [hunk 1]
```diff
@@ -14,9 +14,8 @@
 
 import com.example.domain.Item;
 import com.example.domain.LoginUser;
-
+import com.example.domain.Order;
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:17]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'com.example.domain.Order' import. Should be before 'org.springframework.web.bind.annotation.RequestMapping'.

File: src/main/java/com/example/controller/AdminController.java [hunk 2]
```diff
@@ -96,4 +95,11 @@ public String showUserList(Model model) {
 		model.addAttribute("userList", userList);
 		return "admin/user_list";
 	}
+
+	@GetMapping("/show-order-list")
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:99]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:99]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:99]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocMethodCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/main/java/com/example/controller/AdminController.java [hunk 3]
```diff
@@ -96,4 +95,11 @@ public String showUserList(Model model) {
...
+	public String showOrderList(Model model) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:100]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/AdminController.java [hunk 4]
```diff
@@ -96,4 +95,11 @@ public String showUserList(Model model) {
...
+		List<Order> orderList = adminService.showOrderList();
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:101]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:101]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/AdminController.java [hunk 5]
```diff
@@ -96,4 +95,11 @@ public String showUserList(Model model) {
...
+		model.addAttribute("orderList", orderList);
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:102]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:102]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/AdminController.java [hunk 6]
```diff
@@ -96,4 +95,11 @@ public String showUserList(Model model) {
...
+		return "admin/order_list";
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:103]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:103]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/AdminController.java [hunk 7]
```diff
@@ -96,4 +95,11 @@ public String showUserList(Model model) {
...
+	}
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:104]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:104]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/com/example/repository/OrderRepository.java [hunk 8]
```diff
@@ -194,6 +194,23 @@ public Order load(Integer orderId) {
 		return orderList.get(0);
 	}
 
+	public List<Order> findAll() {
```
Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:197]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:197]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:197]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocMethodCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/main/java/com/example/repository/OrderRepository.java [hunk 9]
```diff
@@ -194,6 +194,23 @@ public Order load(Integer orderId) {
...
+		StringBuilder sb = new StringBuilder();
```
Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:198]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:198]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/repository/OrderRepository.java [hunk 10]
```diff
@@ -194,6 +194,23 @@ public Order load(Integer orderId) {
...
+		sb.append(
```
Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:199]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:199]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/repository/OrderRepository.java [hunk 11]
```diff
@@ -194,6 +194,23 @@ public Order load(Integer orderId) {
...
+				"SELECT o.id as o_id, o.user_id AS o_user_id, o.status AS o_status, o.total_price AS o_total_price, o.order_date AS o_order_date, o.destination_name AS o_destination_name, o.destination_email AS o_destination_email, o.destination_zipcode AS o_destination_zipcode, o.destination_pref AS o_destination_pref, o.destination_municipalities AS o_destination_municipalities, ");
```
Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:200]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 403).

Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:200]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/repository/OrderRepository.java [hunk 12]
```diff
@@ -194,6 +194,23 @@ public Order load(Integer orderId) {
...
+		sb.append(
```
Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:201]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:201]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/repository/OrderRepository.java [hunk 13]
```diff
@@ -194,6 +194,23 @@ public Order load(Integer orderId) {
...
+				"o.destination_address AS o_destination_address, o.destination_tel AS o_destination_tel, o.delivery_time AS o_delivery_time, o.payment_method AS o_payment_method, ");
```
Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:202]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 198).

Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:202]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/repository/OrderRepository.java [hunk 14]
```diff
@@ -194,6 +194,23 @@ public Order load(Integer orderId) {
...
+		sb.append(
```
Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:203]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:203]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/repository/OrderRepository.java [hunk 15]
```diff
@@ -194,6 +194,23 @@ public Order load(Integer orderId) {
...
+				"oItem.id AS oItem_id, oItem.item_id AS oItem_item_id, oItem.order_id AS oItem_order_id, oItem.quantity AS oItem_quantity, oItem.size AS oItem_size, ");
```
Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:204]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 184).

Comment (tkeshun) [inline src/main/java/com/example/repository/OrderRepository.java:204]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

**Review by tkeshun — COMMENTED** [review]
Remaining comments which cannot be posted as a review comment to avoid GitHub Rate Limit

<details>
<summary>checkstyle</summary>

[src/main/java/com/example/repository/OrderRepository.java|205 col 1|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/repository/OrderRepository.java#L205) Line contains a tab character.
[src/main/java/com/example/repository/OrderRepository.java|205 col 17|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/repository/OrderRepository.java#L205) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/com/example/repository/OrderRepository.java|206|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/repository/OrderRepository.java#L206) Line is longer than 100 characters (found 157).
[src/main/java/com/example/repository/OrderRepository.java|206 col 1|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/repository/OrderRepository.java#L206) Line contains a tab character.
[src/main/java/com/example/repository/OrderRepository.java|207|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/repository/OrderRepository.java#L207) Line is longer than 100 characters (found 109).
[src/main/java/com/example/repository/OrderRepository.java|207 col 1|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/repository/OrderRepository.java#L207) Line contains a tab character.
[src/main/java/com/example/repository/OrderRepository.java|207 col 17|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/repository/OrderRepository.java#L207) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/com/example/repository/OrderRepository.java|208|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/repository/OrderRepository.java#L208) Line is longer than 100 characters (found 110).
[src/main/java/com/example/repository/OrderRepository.java|208 col 1|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/repository/OrderRepository.java#L208) Line contains a tab character.
[src/main/java/com/example/repository/OrderRepository.java|208 col 17|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/repository/OrderRepository.java#L208) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/com/example/repository/OrderRepository.java|210 col 1|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/repository/OrderRepository.java#L210) Line contains a tab character.
[src/main/java/com/example/repository/OrderRepository.java|210 col 17|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/repository/OrderRepository.java#L210) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/com/example/repository/OrderRepository.java|211 col 1|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/repository/OrderRepository.java#L211) Line contains a tab character.
[src/main/java/com/example/repository/OrderRepository.java|211 col 17|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/repository/OrderRepository.java#L211) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/com/example/repository/OrderRepository.java|212 col 1|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/repository/OrderRepository.java#L212) Line contains a tab character.
[src/main/java/com/example/repository/OrderRepository.java|212 col 9|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/repository/OrderRepository.java#L212) 'method def rcurly' has incorrect indentation level 8, expected level should be 2.
[src/main/java/com/example/service/AdminService.java|14 col 1|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/service/AdminService.java#L14) Wrong lexicographical order for 'com.example.domain.Order' import. Should be before 'org.springframework.web.multipart.MultipartFile'.
[src/main/java/com/example/service/AdminService.java|93 col 1|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/service/AdminService.java#L93) Line contains a tab character.
[src/main/java/com/example/service/AdminService.java|93 col 9|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/service/AdminService.java#L93) 'method def modifier' has incorrect indentation level 8, expected level should be 2.
[src/main/java/com/example/service/AdminService.java|94 col 1|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/service/AdminService.java#L94) Line contains a tab character.
[src/main/java/com/example/service/AdminService.java|94 col 17|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/service/AdminService.java#L94) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/com/example/service/AdminService.java|95 col 1|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/service/AdminService.java#L95) Line contains a tab character.
[src/main/java/com/example/service/AdminService.java|95 col 17|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/service/AdminService.java#L95) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/com/example/service/AdminService.java|96 col 1|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/service/AdminService.java#L96) Line contains a tab character.
[src/main/java/com/example/service/AdminService.java|96 col 9|](http://github.com/ecsite-project/ec-202304c/blob/7aad0838ffb5ac32e18785085d7fd60d223b29cc/src/main/java/com/example/service/AdminService.java#L96) 'method def rcurly' has incorrect indentation level 8, expected level should be 2.
</details>

**Review by uedamiku — APPROVED** [review]

**Pull request merged**